### ISSUE TITLE
DCGAN loss fixed at initial value

### ISSUE THREAD

**Issue by CoreyJaskolski** [POSTECH-CVLab/PyTorch-StudioGAN#112]
When using the command line command from quick start instructions:
`CUDA_VISIBLE_DEVICES=0,1,2,3 python3 src/main.py -t -e -cfg CONFIG_PATH -data DATA_PATH -save SAVE_PATH`
and the config in src/configs/Tiny_ImageNet/DCGAN.yaml, the Gen_loss start at -0.0 and the Dis_loss at 13.82 remain fixed at this initial value.

The DATA_PATH is the downloaded Tiny_ImageNet data set. The above command just with a new configuration file such as rc/configs/Tiny_ImageNet/WGAN-WC.yaml works as expected with the loss changing and being reported each 100 steps

Logs:
```
[INFO] 2021-11-28 10:56:36 > Step:    100 Progress: 0.1% Elapsed: 0:00:57 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 10:57:27 > Step:    200 Progress: 0.2% Elapsed: 0:01:49 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 10:58:20 > Step:    300 Progress: 0.3% Elapsed: 0:02:42 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 10:59:14 > Step:    400 Progress: 0.4% Elapsed: 0:03:35 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:00:06 > Step:    500 Progress: 0.5% Elapsed: 0:04:27 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:00:58 > Step:    600 Progress: 0.6% Elapsed: 0:05:19 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:01:50 > Step:    700 Progress: 0.7% Elapsed: 0:06:12 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:02:44 > Step:    800 Progress: 0.8% Elapsed: 0:07:05 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:03:36 > Step:    900 Progress: 0.9% Elapsed: 0:07:57 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:04:28 > Step:   1000 Progress: 1.0% Elapsed: 0:08:50 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:05:20 > Step:   1100 Progress: 1.1% Elapsed: 0:09:42 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:06:14 > Step:   1200 Progress: 1.2% Elapsed: 0:10:35 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:07:06 > Step:   1300 Progress: 1.3% Elapsed: 0:11:28 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:07:59 > Step:   1400 Progress: 1.4% Elapsed: 0:12:20 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:08:51 > Step:   1500 Progress: 1.5% Elapsed: 0:13:13 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:09:44 > Step:   1600 Progress: 1.6% Elapsed: 0:14:05 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:10:36 > Step:   1700 Progress: 1.7% Elapsed: 0:14:57 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:11:28 > Step:   1800 Progress: 1.8% Elapsed: 0:15:50 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
[INFO] 2021-11-28 11:12:22 > Step:   1900 Progress: 1.9% Elapsed: 0:16:43 Gen_loss: -0.0 Dis_loss: 13.82 Cls_loss: N/A Topk:  N/A ada_p: N/A
```

**Comment by MINGUKKANG** [comment]
Hello,

I am very sorry for the late reply.

With a lot of work to do lately, I haven't been able to afford to respond to issues raised by StudioGAN users.

I identified that this problem is a typical mode-collapse problem. 

This issue is caused by the change of vanilla gan loss suggested in the issue (https://github.com/POSTECH-CVLab/PyTorch-StudioGAN/issues/67).

I will roll back the vanilla gan loss to the previous verson.

Thank you.

**Comment by MINGUKKANG** [comment]
The rolled-back vanilla loss is slightly different from the vanilla loss presented in the paper, but we rolled back the loss function because it gives a more stable result.

I think the issue has been resolved, so I'm closing the issue.

Thank you.
